python/tests/unit/arcticdb/version_store/test_append.py

```diff
@@ -439,4 +441,150 @@ def test_append_mix_sorted_non_sorted(lmdb_version_store):
     assert df2.index.is_monotonic_increasing == True
     lmdb_version_store.append(symbol, df2)
     info = lmdb_version_store.get_info(symbol)
-    assert info["sorted"] == "UNSORTED"
\ No newline at end of file
+    assert info["sorted"] == "UNSORTED"
+
+
+@use_of_function_scoped_fixtures_in_hypothesis_checked
+@settings(deadline=None, max_examples=10)
+@given(
+    col_per_append_df=st.integers(2, 100),
+    col_name_set=st.integers(1, 10000),
+    num_of_rowss=st.lists(st.lists(st.integers(1, 20), min_size=1, max_size=10), min_size=1, max_size=10),
```

Comment:
Typo: `num_of_rowss`

Reply:
oh wow! can't we just call it `num_rows_per_test_cycle`? I fear everybody else who sees this will just think it's a typo as well unless this is a convention I've never seen before.